app/Domain/Tickets/Templates/showAll.blade.php

```diff
@@ -215,20 +215,19 @@ class="ticketDropdown statusDropdown colorized show {{ $class }} f-left"
                                 </x-global::elements.table.cell>
             
                                 <x-global::elements.table.cell data-order="{{ $milestoneHeadline }}">
-                                    <x-global::forms._archive.dropdownPill
-                                        class="ticketDropdown milestoneDropdown colorized show label-default milestone f-left"
-                                        id="{{ $milestoneDropdownId }}" 
-                                        :style="'background-color:' . $milestoneColor" 
+                                    <x-global::forms.chip
+                                        class="ticketDropdown milestoneDropdown colorized milestone show f-left"
+                                        :bgColor="'background-color:' . $milestoneColor" 
                                         :labelText="$milestoneHeadline" 
                                         type="milestone"
-                                        :parentId="$row['id']">
-                                        <x-slot name="buttonText">
-                                            {{ $milestoneHeadline }} <i class="fa fa-caret-down" aria-hidden="true"></i>
-                                        </x-slot>
-            
-                                        <x-global::actions.dropdown.item class="nav-header border">
+                                        :parentId="$row['id']"
+                                        :quickaddOption="true"
+                                        quickaddPostUrl="{{ BASE_URL }}/hx/tickets/Milestones"
```

Comment:
You could just pass the hx-* attributes rather than create a new attribute. 
Basically just use hx-post and create the appropriate hxPost variable in your component. Maybe add hx-swap, hx-target, hx-trigger etc as well I am pretty sure you could just use the $attributes array that is available in the component and look for entries that start with hx